src/pages/Routes.jsx

```diff
@@ -5,6 +5,8 @@ import Landing from './Landing';
 import SigninComponent from './Signin';
 import SignupComponent from './Signup';
 import NavBar from '../components/NavBar';
+import ResetPassword from '../components/resetPassword';
```

Comment:
Components that will be used in the Route component must reside in the pages folder